### PR TITLE
ahmed/emb-22-bankgreenteam-page-implemented-in-prismic

### PR DESCRIPTION
**Pull request by ahmedlamidi**
This is just the migration of the team members portion of the teams page into prismic.
To test : go to prismic team page, then click preview, open up the code in your IDE and then npm i and npm run dev.
The changes I made are all in the teammember  slice and the team page.

### CODE REVIEW

**Review by brugo-dev — COMMENTED** [review]
Tested locally from your branch.  The team page appears to render and function correctly at various screen sizes and emulated mobile.  It did notice that there might be a slight font-size change, the text renders larger in the new page than the non-prismic page - maybe double check that this is expected. 

The Social Sharer components render properly but I don't think it is using the socialsharerslice. Same with the ThanksSlice, since I don't know where/if it's used I can't test.  Please double check that these should be part of this PR at all - seems like they might belong the migration PR.

Because I'm relatively new to Slices. I recommend trying to get a second reviewer if you can.

File: slices/SocialSharerSlice/index.vue
Comment (brugo-dev) [inline slices/SocialSharerSlice/index.vue:18]: Is SocialSharerSlice supposed to be in this PR? I don't think it is used.

Comment (ahmedlamidi) [reply slices/SocialSharerSlice/index.vue:18]: I had to bring them over to keep the slice machine working properly on this branch. All slices would have to be in all branches and kept updated regardless of if we use them or not.

**Comment by ahmedlamidi** [comment]
Concerning the font difference , I used the same class that was used to make it in the non prismic change. I can check on my end and check what is causing the issue.

**Comment by Chris33871** [comment]
Unable to fully test this since I can't run dev but the code looks good.
Are you sure you wish to create the index.ts file? Since it's not present on the main branch it might be useless. It's worth checking if deleting it results in any problems. I think it's best to stick with the current setup that doesn't include the .ts file.

**Review by sopoour — APPROVED** [review]
I've just tested it and looked through the code. Looks all good to me. Just have one small comment that gives me a lint warning. 

Also you need to merge with main and resolve the conflicts before merging

**Pull request merged**
